modules/trusted-firmware-m/Kconfig

```diff
@@ -64,12 +64,41 @@ config TFM_KEY_FILE_NS
 	  firmware images.
 
 config TFM_PROFILE
-	string "The build profile used for TFM Secure image."
+	string
 	depends on BUILD_WITH_TFM
+	default "profile_small" if TFM_PROFILE_TYPE_SMALL
+	default "profile_medium" if TFM_PROFILE_TYPE_MEDIUM
+	default "profile_large" if TFM_PROFILE_TYPE_LARGE
 	help
 	  Build profile used to build tfm_s image. The available values are
-	  profile_medium and profile_small. The default profile does not need
-	  to have this configuration set.
+	  profile_large, profile_medium and profile_small. The default profile
```

Comment:
Setting this config via menuconfig doesn't really work well. Should that be mentioned here?

Reply:
With the conditional removal of the prompt, i don't think we need to mention anything.